### DISCUSSION TITLE
offline build Doc

### DISCUSSION THREAD

**Discussion by kingstar158** [facebookincubator/velox#6974] [Q&A]
Are there any guidelines for offline build ？？

**Comment by ViggoC** [comment]
@kingstar158 What do you mean "offline build", build in an environment with out network?

**Comment by Yohahaha** [comment]
download deps to local, then set VELOX_DEPS_URL env variables.
for example, offline build gtest:
1. download https://github.com/google/googletest/archive/refs/tags/v1.13.0.tar.gz to /opt/
2. export VELOX_GTEST_URL=/opt/googletest-1.13.0.tar.gz
3. just start build.
